### PR TITLE
Bookstore: Styling

### PR DESCRIPTION
**Pull request by WilhelmK109**
### In this milestone, I did the following:
- Added CSS code to style the bookstore page
- Added missing JSX elements to the project

### CODE REVIEW

**Review by t-yanick — CHANGES_REQUESTED** [review]
# STATUS: CHANGES REQUIRED

Hi @WilhelmK109 , 

### Good job so far!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  Good commit messages
:heavy_check_mark: Descriptive Readme file

There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/AddBookButton.js [hunk 1]
```diff
@@ -28,7 +28,7 @@ export default function AddBookButton() {
   return (
     <div className="form-container">
       <h2>Add New Book</h2>
-      <form onSubmit={handleAddBook}>
+      <form onSubmit={handleAddBook} className="add-form">
         <input className="input-title" type="text" value={title} onChange={(e) => setTitle(e.target.value)} placeholder="Book title" />
         <input className="input-author" type="text" value={author} onChange={(e) => setAuthor(e.target.value)} placeholder="Author" />
```
[the remark this answers is not in the document]
Comment (WilhelmK109) [reply src/components/AddBookButton.js:33]: @t-yanick Thank you sir, but how about the instruction below, or is it no longer applicable?
Screenshot (92)

[the remark this answers is not in the document]
Comment (t-yanick) [reply src/components/AddBookButton.js:33]: Hi , it definitely applies.  I missed that. You can ignore that required change.  Thanks

**Review by KayLemba — APPROVED** [review]
[[# STATUS: APPROVED :green_circle: :clap: 

Hello @WilhelmK109       , 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉  Keep up the good work 🥂 

HIGHLIGHTS:

- [x]  Your repo is professional  😎 
- [x] Descriptive P.R 💯 
- [x] linters are passing
- [x] Professional README 💯 
- [x] Nice UI design 🥇 
 










 








Now you can merge this branch and move on :1st_place_medal: 
Happy Coding cheers 🥇 💯 



well-1

Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._]]

File: src/components/AddBookButton.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (WilhelmK109) [reply src/components/AddBookButton.js:33]: 👍

**Pull request merged**